v2/refs/convert.go

```diff
@@ -76,6 +76,34 @@ func ObjectIDFromGRPCMessage(m *refs.ObjectID) *ObjectID {
 	return o
 }
 
+func ObjectIDListToGRPCMessage(ids []*ObjectID) []*refs.ObjectID {
+	if ids == nil {
+		return nil
+	}
+
+	idsV2 := make([]*refs.ObjectID, 0, len(ids))
+
+	for i := range ids {
+		idsV2 = append(idsV2, ObjectIDToGRPCMessage(ids[i]))
+	}
+
+	return idsV2
+}
+
+func ObjectIDListFromGRPCMessage(idsV2 []*refs.ObjectID) []*ObjectID {
```

Comment:
Can we reuse it in `SearchResponseBodyToGRPCMessage` and `SearchResponseBodyFromGRPCMessage`?

Reply:
Yes we can